test/http-request.test.js

```diff
@@ -2,20 +2,34 @@ import { fixture, expect, aTimeout } from '@open-wc/testing';
 
 import '../src/custom-element.js';
 import '../src/http-request.js';
-import defaults, {HttpRequestSimple} from "../stories/http-request.stories";
+import defaults, {Demo, LifecycleInitialized} from "../stories/http-request.stories";
+import "../src/mocks/browser.js";
+
 
 const defs = {}; Object.keys(defaults.argTypes).map( k=> defs[k]= defaults.argTypes[k].defaultValue);
 const renderStory = async (story) => fixture( story({ ...defs, ...story.args }) );
 
 describe('http-request', () =>
 {
-    it('http-request init', async () =>
+    it('http-request happy path', async () =>
     {
-        const el = await renderStory(HttpRequestSimple);
+        const el = await renderStory(Demo);
```

Comment:
The 💎 of TDD is reusing the StoryBook items in the unit tests. 

Visual development is started in StoryBook and checked visually by Chromatic StoryBook integration and functionally by unit test.